### ISSUE TITLE
MUX ProgressRing Causes Exception during Runtime

### ISSUE THREAD

**Issue by robloo** [unoplatform/uno#5877]
## Current behavior

Has anyone tested the MUX ProgressRing recently? I can't get it to work with Uno 3.6.6 or 3.7.3. At first it fails to build:

 > Could not find 1 Android X assemblies, make sure to install the following NuGet packages:
Xamarin.AndroidX.AppCompat.AppCompatResources
You can also copy-and-paste the following snippet into your .csproj file:
&lt;PackageReference Include="Xamarin.AndroidX.AppCompat.AppCompatResources" Version="1.2.0.5" /&gt; <app_name>.Droid

However, even after adding `Xamarin.AndroidX.AppCompat.AppCompatResources` and `Uno.UI.Lottie` the app fails to startup (it compiles though). It gives the exception:

 > Java.Lang.NoSuchFieldError
  Message=No static field abc_vector_test of type I in class Landroidx/appcompat/resources/R$drawable; or its superclasses (declaration of 'androidx.appcompat.resources.R$drawable' appears in /data/app/<app_name>-gTA26OTkGqsU2uDTp14pHA==/base.apk)

## Expected behavior

MUX ProgressRing using lottie animations should work.

## Workaround

Use the built-in ProgressRing. As the Windows 10 ProgressRing has bugs the MUX version does not, conditional XAML is a good idea.

## Environment

Nuget Package: 

- [X] Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia
- [ ] Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia
- [ ] Uno.SourceGenerationTasks
- [ ] Uno.UI.RemoteControl / Uno.WinUI.RemoteControl
- [ ] Other: 

Nuget Package Version(s): 3.6.6 / 3.7.3

Affected platform(s):

- [ ] iOS
- [x] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

IDE:

- [ ] Visual Studio 2017 (version: )
- [x] Visual Studio 2019 (version: )
- [ ] Visual Studio for Mac (version: )
- [ ] Rider Windows (version: )
- [ ] Rider macOS (version: )
- [ ] Visual Studio Code (version: )

Relevant plugins:

- [ ] Resharper (version: )

**Comment by jeromelaban** [comment]
Thanks for the report. The progress ring is used in different applications build daily. This may be a dependency conflict in the application you're building.

**Comment by robloo** [comment]
@jeromelaban 

>The progress ring is used in different applications built daily.

Thanks for confirming this is working elsewhere. I expect you're right and the problem is on my end.

> This may be a dependency conflict in the application you're building.

Dependencies are minimal and I can't see any conflict here. The issue does not occur until I add `Uno.UI.Lottie` which then requires I add `Xamarin.AndroidX.AppCompat.AppCompatResources` to build. The list of dependencies is below (excluding the ones already mentioned).

```xml
<ItemGroup>
    <PackageReference Include="Microsoft.Bcl.HashCode">
      <Version>1.1.1</Version>
    </PackageReference>
    <PackageReference Include="Microsoft.Xml.SgmlReader">
      <Version>1.8.21</Version>
    </PackageReference>
    <PackageReference Include="NodaTime">
      <Version>3.0.5</Version>
    </PackageReference>
    <PackageReference Include="SkiaSharp">
      <Version>2.80.2</Version>
    </PackageReference>
    <PackageReference Include="SkiaSharp.Views.Uno">
      <Version>2.80.2</Version>
    </PackageReference>
    <PackageReference Include="System.Text.Encoding.CodePages">
      <Version>5.0.0</Version>
    </PackageReference>
    <PackageReference Include="Uno.Microsoft.Toolkit.Uwp.UI.Controls">
      <Version>7.0.0</Version>
    </PackageReference>
    <PackageReference Include="Uno.Microsoft.Toolkit.Uwp.UI.Controls.DataGrid">
      <Version>7.0.0</Version>
    </PackageReference>
    <!-- Xamarin Required -->
    <PackageReference Include="System.Data.Common">
      <Version>4.3.0</Version>
    </PackageReference>
    <!-- Uno Required -->
    <PackageReference Include="Uno.UI" Version="3.6.6" />
    <PackageReference Include="Uno.UI.Maps" Version="3.6.6" />
    <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.3.0.1" />
    <PackageReference Include="Uno.UI.RemoteControl" Version="3.6.6" Condition="'$(Configuration)'=='Debug'" />
    <PackageReference Include="Uno.UniversalImageLoader" Version="1.9.33" />
    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="5.0.0" />
    <PackageReference Include="Microsoft.Extensions.Logging.Filter" Version="1.1.2" />
  </ItemGroup>
```

**Comment by jeromelaban** [comment]
Thanks. Are you able to reproduce this using a sample app, or is it only in your app ?

**Comment by robloo** [comment]
> Are you able to reproduce this using a sample app, or is it only in your app ?

Only in a real app so far, in a simple repro app it works. However, I do still have to add `Xamarin.AndroidX.AppCompat.AppCompatResources` so the docs appear to need an update regardless.

The XAML and code-behind in the real app is complex and I don't have an idea where to start looking. Unfortunately, this is very low priority and I've already worked around this issue with conditional XAML. I only ran across this as I stopped using the UWP toolkit Loading control. As you confirmed it works in other apps, and it's working in the simple repro app I just made, I can close this if you want. If I don't have the time to investigate further I can't expect you too. 

That said, this doesn't appear to be dependency-related at this time.
